src/monitor.rs

```diff
@@ -68,12 +68,10 @@ impl VideoMode {
         self.video_mode.bit_depth()
     }
 
-    /// Returns the refresh rate of this video mode. **Note**: the returned
-    /// refresh rate is an integer approximation, and you shouldn't rely on this
-    /// value to be exact.
+    /// Returns the refresh rate of this video mode in mHz.
     #[inline]
-    pub fn refresh_rate(&self) -> u16 {
-        self.video_mode.refresh_rate()
+    pub fn refresh_rate_mhz(&self) -> u32 {
```

Comment:
`mhz` could easily be mistaken as `MHz`... sigh. This is a case where I don't (personally) think standard naming conventions should apply.

Reply:
Writing it out as `refresh_rate_millihertz` would remove that ambiguity. I'm not that attached to the shortened name myself.

Reply:
renamed.